CMSIS/Core/Include/cmsis_armcc.h

```diff
@@ -110,6 +110,10 @@
 
 /* #########################  Startup and Lowlevel Init  ######################## */
 
+#ifndef __EARLY_INIT
+#define __EARLY_INIT()
```

Comment:
Why did you add the `__EARLY_INIT()` provision to the compiler headers? Just to have them empty by default? Or do you expect the implementation to be compiler specific?